node-services/cards-external-diffusion/src/cardsExternalDiffusion.ts

```diff
@@ -0,0 +1,159 @@
+/* Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * See AUTHORS.txt
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+import express from 'express';
+import bodyParser from 'body-parser';
+import config from 'config';
+
+import logger from './domain/server-side/logger';
+
+import AuthenticationService from './domain/client-side/authenticationService';
+import SendMailService from './domain/client-side/sendMailService';
+import EventBusListener from './domain/client-side/eventBusListener';
+import CardsDiffusionControl from './domain/application/cardsDiffusionControl';
+import OpfabInterface from './domain/server-side/opfabInterface';
+import CardsExternalDiffusionService from './domain/client-side/cardsExternalDiffusionService';
+import ConfigService from './domain/client-side/configService';
+
+const app = express();
+app.use(bodyParser.json());
+
+app.use(express.static("public"));
+const adminPort = config.get('adminPort');
+
+
+const activeOnStartUp = config.get('activeOnStartup');
+
+const configService = new ConfigService(config.get('defaultConfig'), 'config/serviceConfig.json', logger);
+
+
+
+const authenticationService = new AuthenticationService()
+    .setTokenExpireClaim(config.get('opfab.tokenExpireClaim'))
+    .setLogger(logger);
+
+
+const mailService = new SendMailService(config.get('mail'));
+    
+const opfabInterface = new OpfabInterface()
+    .setLogin(config.get('opfab.login'))
+    .setPassword(config.get('opfab.password'))
+    .setOpfabUrl(config.get('opfab.url'))
+    .setOpfabGetUsersUrl(config.get('opfab.usersUrl'))
+    .setOpfabGetUsersConnectedUrl(config.get('opfab.connectedUsersUrl'))
+    .setOpfabGetCardsUrl(config.get('opfab.cardsUrl'))
+    .setOpfabGetTokenUrl(config.get('opfab.getTokenUrl'))
+    .setAuthenticationService(authenticationService)
+    .setLogger(logger);
+
+opfabInterface.loadUsersData();
+
+const listener = new EventBusListener();
```

Comment:
What about moving the EventbusListener into CardsExternalDiffusionService  instead ? Because it has to be started and stopped as the service and I see OpfabInterface just as an interface to other services.